schemas/v1.0/schema.json

```diff
@@ -67,6 +67,12 @@
                         "replace",
                         "ignore"
                     ]
+                },
+                "xrefResolver": {
+                    "type": "string",
+                    "description": "Format: `uid,{propertyName1},{propertyName2},...,{propertyNameN}`. It MUST start with property name`uid`. Other `propertyName`s are joined by `,`. `propertyName`s are the property names of current object. It defines the properties of current xref object, which is used when others cross reference it.",
+                    "default": "uid,name",
+                    "pattern": "^uid[,$]"
```

Comment:
>^uid[,$] [](start = 32, length = 8)

^uid(,|$)

Reply:
oops

---
In reply to: [135698008](https://github.com/dotnet/docfx/pull/2008#discussion_r135698008) [](ancestors = 135698008)